data/data.json

```diff
@@ -9499,6 +9499,49 @@
    "aciGbpServerContainer": "noiro/gbp-server:5.1.1.0.1ae238a",
    "aciOpflexServerContainer": "noiro/opflex-server:5.1.1.0.1ae238a"
   },
+  "v1.21.14-rancher1-1": {
+   "etcd": "rancher/mirrored-coreos-etcd:v3.4.16-rancher1",
+   "alpine": "rancher/rke-tools:v0.1.80",
```

Comment:
@dhruvmewada15 Could you bump the `rke-tools` version to `v0.1.83` for all new versions? This is the latest version and should include fixes for issues such as https://github.com/rancher/rancher/issues/37639.

Reply:
@jakefhyde Updated the `rke-tools` version to `v0.1.83`